swap/swap.go

```diff
@@ -437,50 +451,57 @@ func (s *Swap) getContractOwner(ctx context.Context, address common.Address) (co
 }
 
 // StartChequebook deploys a new instance of a chequebook if chequebookAddr is empty, otherwise it wil bind to an existing instance
-func (s *Swap) StartChequebook(chequebookAddr common.Address) error {
-	if chequebookAddr != (common.Address{}) {
-		if err := s.BindToContractAt(chequebookAddr); err != nil {
-			return err
+func (s *Swap) StartChequebook(chequebookAddrFlag common.Address) error {
+
+	toUseChequebook, err := s.loadChequebook()
```

Comment:
for me, `toUseChequebook` sounds like a boolean variable. consider naming it `chequebookToUse` if the variable is—conceptually—a chequebook

Reply:
Renamed